go/chat/convsource.go

```diff
@@ -1066,13 +1066,31 @@ func (s *HybridConversationSource) expungeNotify(ctx context.Context, uid gregor
 	}
 }
 
+// notifyReactionDeletes notifies the GUI after reactions are deleted
+func (s *HybridConversationSource) notifyReactionDeletes(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID, reactionDeletes []chat1.ReactionDelete) {
+	s.Debug(ctx, "notifyReactionDeletes: %d deletions", len(reactionDeletes))
+	if len(reactionDeletes) > 0 {
+		topicType := chat1.TopicType_NONE
```

Comment:
We don't know the `topicType` here?

Reply:
we could hardcode to `CHAT` but i thought this was just a future safe thing since we do the same for ephemeral notify https://github.com/keybase/client/pull/12830/files#diff-cf8add58c26f1cbee6e6e1e5f4a20d6eR1092

Reply:
Oh wait this is correct, sorry!